catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/WorkspaceTransformer.java

```diff
@@ -277,4 +292,38 @@ public Metacard toMetacardFromXml(Serializable xml) {
 
     return null;
   }
+
+  public void addListActions(Metacard workspaceMetacard, Map<String, Object> workspaceAsMap) {
```

Comment:
✏️ addListActionsToWorkspaceAsMap might be a little more clear as to what this method is doing

Reply:
This has been changed since #3211 was merged and is now called something else but has a comment explaining it.